CMakeLists.txt

```diff
@@ -357,3 +346,30 @@ if(${STRINGZILLA_BUILD_SHARED})
 
 
 endif()
+
+if(STRINGZILLA_INSTALL)
+  install(
+    TARGETS stringzilla_shared
+    ARCHIVE
+    BUNDLE CXX_MODULES_BMI
```

Comment:
We don't need CXX BMI here as well, I believe.

Reply:
I understand for SimSIMD as it is written in C but StringZilla is a C++ library. I would install everything that is related to C++ for a C++ library.

Reply:
Not really, StringZilla in-core is two C99 libraries: header-only and pre-compiled shared lib with dynamic dispatch. There is also a third header-only C++11 wrapper, so we need to install them all with different settings. But this requires a full CMake rewrite and will take much more time.

It's expected with the next major release.

Reply:
Yes the CMake files really need a revamp. I would suggestall projects SimSIMD, StringZilla and usearch.

Reply:
Stille remove `CXX_MODULES_BMI`?

Reply:
Yes, let's remove the BMI for now, and keep the bigger changes for future. If you have broader changes in mind, can you please comment on this [PR](https://github.com/ashvardanian/StringZilla/pull/85)?